### PR TITLE
Refactor/markup header

### PR DESCRIPTION
**Pull request by foxriver660**
-add UI kit system(TS)
-add indentation system(5x)/box.css in vendor folder

### CODE REVIEW

File: src/components/UI/icons/NotificationIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React, { FC } from "react";
+
+type TNotificationIcon = {type: 'default' | 'active'}
```
[the remark this answers is not in the document]
Comment (foxriver660) [reply src/components/UI/icons/NotificationIcon.tsx:3]: done

File: src/components/UI/index.js
[the remark this answers is not in the document]
Comment (foxriver660) [reply src/components/UI/index.js:1]: done

File: src/components/UI/icons/NotificationIcon.tsx
[the remark this answers is not in the document]
Comment (foxriver660) [reply src/components/UI/icons/NotificationIcon.tsx:1]: done

File: src/components/UI/icons/ProfileIcon.tsx
[the remark this answers is not in the document]
Comment (foxriver660) [reply src/components/UI/icons/ProfileIcon.tsx:1]: done

File: src/components/header/header.jsx [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
-import Logo from "../logo/logo";
-
 
 export default function Header() {
   return (
     <header className={style.header}>
-      <div className={style.container}>
-        <div className={style.nav_wrapper}>
+      <div className={style.nav_wrapper}>
+        <div className={style.nav_main}>
```
[the remark this answers is not in the document]
Comment (foxriver660) [reply src/components/header/header.jsx:9]: done

**Review by BKonstantine — APPROVED** [review]
Отлично!

**Pull request merged**